Automattic/jetpack
# Contact Form: Adding non minified JS file to ensure there is a choice of non-minified and minified

Fixes https://github.com/Automattic/vulcan/issues/316

## Proposed changes:

* ~This PR adds the file path to the non-minified version of `tinymce-plugin-form-button.js`, into `Assets::get_file_url_for_environment` within `mce_external_plugins`.~ Previously both minified versions (located at `jetpack_vendor/automattic/jetpack-forms/dist/contact-form/js/tinymce-plugin-form-button.js`) had been added to `Assets::get_file_url_for_environment`, as a quick fix to prevent an issue caused by calling the minified JS file using `.min.js`, however that file path does not exist in the package version of the contact form.
* After a discussion on Slack, this PR is now using `plugins_url` and only using the minified file version. Slack ref: p1715614401962229-slack-CBG1CP4EN
* Other options included ensuring the minified version name was changed to include `min`, however that didn't seem necessary.


### Other information:


## Jetpack product discussion

https://github.com/Automattic/vulcan/issues/316

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To test, apply this PR and then install and activate the Classic Editor plugin.
* Build the assets with `jetpack build packages/forms --production`
* Create a new post draft, and use the 'Add Contact Form' button to add a new form (or open the editor for a post that already has a form added using this method). It should add with no issues.
* There should also be no issues on WoA sites, nor simple sites, with this patch.
